CHANGELOG.md

```diff
@@ -7,7 +7,24 @@
 - Enables controller-runtime metrics in Helm operator projects. ([#1482](https://github.com/operator-framework/operator-sdk/pull/1482))
 
 ### Changed
+
 - Remove TypeMeta declaration from the implementation of the objects [#1462](https://github.com/operator-framework/operator-sdk/pull/1462/)
+- Upgrade [`controller-runtime`](https://github.com/kubernetes-sigs/controller-runtime) version from `v0.1.10` to `v0.2.0-beta.2`
+- Upgrade Kubernetes version from `kubernetes-1.13.1` to `kubernetes-1.14.1`
+- Upgrade `github.com/operator-framework/operator-lifecycle-manager` version from `b8a4faf68e36feb6d99a6aec623b405e587b17b1` to `0.10.1`
+
+### Breaking changes
+
+- The package `sigs.k8s.io/controller-runtime/pkg/runtime/scheme` is deprecated. Replace this import with `sigs.k8s.io/controller-runtime/pkg/scheme` where relevant.
```

Comment:
It looks like the old `pkg/runtime/scheme` package exists, but no longer has any contents. Should we just say it's removed?

```suggestion
- The package `sigs.k8s.io/controller-runtime/pkg/runtime/scheme` is removed. Replace this import with `sigs.k8s.io/controller-runtime/pkg/scheme` where relevant.
```

Reply:
Technically it is [deprecated](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/runtime/scheme/scheme.go#L21) and still exists, but contains nothing. I'll add a note that package code has been removed.